packages/manager/modules/manager-pci-common/src/components/logs/StreamSubscriptions.component.tsx

```diff
@@ -183,16 +158,9 @@ export function StreamSubscriptions({
               }
             }
           }}
-          disabled={isCreationPending ? true : undefined}
-          inline
-        >
-          {t('list_button_subscribe')}
-          {isCreationPending && (
-            <span slot="start">
-              <OsdsSpinner className="mt-3" size={ODS_SPINNER_SIZE.sm} inline />
-            </span>
-          )}
-        </OsdsButton>
+          isLoading={isCreationPending}
+          isDisabled={isCreationPending ? true : undefined}
```

Comment:
Think this issue of being either true or undefined was fixed in ods18 if am not wrong ! 
https://ovh.github.io/design-system/latest/?path=/docs/ods-components-button--technical-information

Reply:
@seven-amid Yep, this is absolutely correct, I'll push the change

Reply:
@seven-amid fixed